### PR TITLE
Fixes #487 Made image zoomable after clicking on the image

### LINKED ISSUE #487: feature request: make images zoomable
after clicking on an image it can't be zoomed in on which is unhelpful on a 5-6" screen and a full screen, full HD screenshot for example.

### PR DESCRIPTION
**Pull request by Ramkishorevit**
**Summary of changes**

Added a new java View class called TouchImageView using which the gestures for zooming in and zooming out were configured. Then while setting the target in Glide it is set to the new TouchImageView rather than normal ImageView

18596277_1553741207971979_757172210_o
18641352_1553741307971969_1965190982_o

Please make sure these boxes are checked before submitting your pull request - thanks! [Guide](./PULL_REQUEST_GUIDE.md)

- [x] Code is formatted.

- [x] Run the lint tests with `./gradlew lintDebug` and make sure BUILD is SUCCESSFUL

- [x] If new feature is implemeted then it is compatible with Night theme too.

- [x] Commit messages are well-written.

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@Ramkishorevit, it looks like you've already signed the Dropbox CLA. Thanks!

**Comment by saketkumar95** [comment]
@Ramkishorevit Thanks for working on this. Can you please provide the short video of all the corner cases like zooming of GIFs and images and also mark the pixels of each image specially in gifs.

**Comment by Ramkishorevit** [comment]
@saketkumar95 

The pixel size of the first image is 1352 * 760  (height*width)

The pixel size of the second image which is a GIF is 216 * 230 (height*width)

Please find the video in the attachment


[device-2017-05-19-162623.mp4.zip](https://github.com/zulip/zulip-android/files/1013963/device-2017-05-19-162623.mp4.zip)

**Comment by Ramkishorevit** [comment]
@saketkumar95 Any changes required here?

**Comment by Ramkishorevit** [comment]
@saketkumar95 I have fixed it using an external library as you said.

**Comment by kunall17** [comment]
Maybe we can use a more updated (worked on) library like https://github.com/chrisbanes/PhotoView ?

**Comment by Ramkishorevit** [comment]
@kunall17 @saketkumar95  Fixed using [https://github.com/chrisbanes/PhotoView](https://github.com/chrisbanes/PhotoView) . Kindly review

File: app/src/main/res/layout/activity_photo_view.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
         android:visibility="visible" />
 
 
-    <ImageView
+    <uk.co.senab.photoview.PhotoView
```
Comment (kunall17) [inline app/src/main/res/layout/activity_photo_view.xml]: Works well, but what's with this name` uk.co.senab.photoview.PhotoView` shouldn't it be com.github.chrisbanes:PhotoView ?

Comment (Ramkishorevit) [inline app/src/main/res/layout/activity_photo_view.xml]: Ya I will change it. Someother dependency should've been already present in the app that has additional feature of zooming then.

Comment (Ramkishorevit) [inline app/src/main/res/layout/activity_photo_view.xml]: From the version 1.3 they have updated the internal package name. That is why

**Comment by kunall17** [comment]
Merged, thanks @Ramkishorevit 👍

**Pull request closed without merging**